src/framework/mpas_io.F

```diff
@@ -1847,6 +1849,13 @@ subroutine MPAS_io_get_var_generic(handle, fieldname, intVal, intArray1d, intArr
       character (len=:), pointer :: charVal_p
       character (len=:), dimension(:), pointer :: charArray1d_p
 
+      ! local variables returned from MPAS_io_inq_var
+      integer :: fieldtype
+      integer :: ndims
+      integer, dimension(:), pointer :: dimsizes
+      character (len=StrKIND), dimension(:), pointer :: dimnames
+      character (len=StrKind) :: message
```

Comment:
For consistency with the line above it and elsewhere in MPAS, let's change `StrKind` to `StrKIND` on this line.